hackathon_site/hardware/test_api.py

```diff
@@ -400,6 +400,18 @@ def test_team_id_filter(self):
         returned_ids = [res["id"] for res in results]
         self.assertCountEqual(returned_ids, [1])
 
+    def test_name_search_filter(self):
+        self._login()
+
+        url = self._build_filter_url(name="name")
```

Comment:
For the search filter, it's just `<url>?search=whatever`, and that will search all of the search fields you specify.

This test is actually a false-positive - since you have no filter arguments that look for `?name`, nothing happens to the results. But since there's only one `Incident` object created, this test doesn't capture that - and technically, none of the other ones do either.

Can you add a second `Incident` object to this test class? Your hardware filter test will be good to go then, and if you attach the incident to a different team (aka new order and order item as well), your team filter test will be good as well.